README.md

```diff
@@ -130,6 +130,16 @@ The test suite requires a PostgreSQL database, along with the following environm
 > tox -e unit,integration -p
 > ```
 
+### Executing flows against a local Metadata service
+
+With the metadata service up and running at `http://localhost:8080`, you are able to use this as the service when executing Flows with the Metaflow client locally via
+
+```sh
+METAFLOW_SERVICE_URL=http://localhost:8080 python3 basicflow.py run
```

Comment:
Do we also need to include `--metadata=service` parameter here?

Reply:
good catch, added it as well.